Sources/Screens/SignOut/SignOutSuccessfulViewModel.swift

```diff
@@ -0,0 +1,34 @@
+import GDSAnalytics
+import GDSCommon
+import Logging
+import UIKit
+
+struct SignOutSuccessfulViewModel: GDSCentreAlignedViewModel,
+                                   GDSCentreAlignedViewModelWithPrimaryButton,
+                                   BaseViewModel {
+    let title: GDSLocalisedString = "app_signedOutTitle"
+    var body: GDSLocalisedString?
+    let primaryButtonViewModel: ButtonViewModel
+    
+    let analyticsService: OneLoginAnalyticsService
+    
+    let rightBarButtonTitle: GDSLocalisedString? = nil
+    let backButtonIsHidden: Bool = true
+    
+    init(analyticsService: OneLoginAnalyticsService,
+         withWallet: Bool = false,
+         buttonAction: @escaping () -> Void) {
+        self.analyticsService = analyticsService
+        self.body = withWallet ? "app_signedOutBodyWithWallet" : "app_signedOutBodyNoWallet"
+        self.primaryButtonViewModel = AnalyticsButtonViewModel(titleKey: "app_continueButton",
+                                                               shouldLoadOnTap: true,
+                                                               analyticsService: analyticsService) {
+            buttonAction()
+        }
+       
+    }
```

Comment:
Do we need to pass in this `withWallet` argument? Could maybe just have:
```suggestion
    init(analyticsService: OneLoginAnalyticsService,
         buttonAction: @escaping () -> Void) {
        self.analyticsService = analyticsService
        self.body = WalletAvailabilityService.hasAccessedBefore ? "app_signedOutBodyWithWallet" : "app_signedOutBodyNoWallet"
        self.primaryButtonViewModel = AnalyticsButtonViewModel(titleKey: "app_continueButton",
                                                               shouldLoadOnTap: true,
                                                               analyticsService: analyticsService) {
            buttonAction()
        }
    }
```